app/controllers/staff_info_controller.rb

```diff
@@ -0,0 +1,66 @@
+class StaffInfoController < ApplicationController
+  before_action :authorize_prison_request
+
+  STAFF_INFO_ENDPOINT = ENV.fetch('STAFF_INFO_ENDPOINT', nil)
+
+  before_action :redirect_to_pdf, if: :pdf?
+  before_action :fetch_nav, only: :show
+  before_action :proxy_content, only: :show
+
+  def index
+    @content = proxy_content
+    render :index, status: req.status
+  end
+
+  def show
+    @content = markdown? ? convert_markdown : proxy_content
+    render :show, status: req.status
```

Comment:
If the page is not found in S3 it will show a blank page, should we be catching error codes differently to render our own error pages? I think as a user it would be surprising to see a page with no content.